### DISCUSSION TITLE
The Percentage shown at the overview page is not the pass percentage in v2.29

### DISCUSSION THREAD

**Discussion by Ravi-Prasanth-122** [allure-framework/allure2#2605] [Bug Reports] [answered]
When i generate the allure report in v2.29 , the percentage shown at the overview page is not pass percentage . Instead , some other percentage is shown which i am not able to derive at the calculation of it . 
Whereas , when the same is generated from v2.18 , the pass percentage is shown in the overview page . 

Report Generated from allure version 2.29 (90.22%): 
<img width="957" alt="Screenshot 2024-06-19 at 9 50 06 PM" src="https://github.com/allure-framework/allure2/assets/124162688/0b836168-c7d7-479e-a322-91d94924af7a">
Pass percent (80.59%): 
<img width="966" alt="Screenshot 2024-06-19 at 9 49 56 PM" src="https://github.com/allure-framework/allure2/assets/124162688/2f8a6e28-3806-421b-b0d7-c92debe98f1e">

Report Generated from allure version 2.18 (80.59%) : 
<img width="951" alt="Screenshot 2024-06-19 at 9 52 04 PM" src="https://github.com/allure-framework/allure2/assets/124162688/6dfd371f-1dfc-4324-832f-04dece22ca39">
Pass percent (80.59%): 
<img width="973" alt="Screenshot 2024-06-19 at 9 52 11 PM" src="https://github.com/allure-framework/allure2/assets/124162688/b8e486c0-3646-4f1a-9345-28fb5d41a761">

**Comment by baev** [comment, marked as answer]
Skipped tests are now excluded from the pass rate calculation

**Reply by Ravi-Prasanth-122** [reply]
Thanks @baev .

**Reply by PePeLM** [reply]
@baev Any idea when that will be fixed? I'm generating report with allure-jenkins-plugin@2.34.0.
And still wrong % shown.
<img width="1303" height="413" alt="image" src="https://github.com/user-attachments/assets/04953995-db8e-478e-9be5-1d7e6a9c1b65" />
